http.go

```diff
@@ -276,7 +282,10 @@ func httpPutReputation(w http.ResponseWriter, r *http.Request) {
 	}
 	exc := false
 	if rep.Type == "ip" {
-		exc = isException(rep.Object)
+		exc, err = isException(rep.Object)
```

Comment:
in the case where `err` is non-nil here, is `exc` always going to be false (guessing that is the case?)

Reply:
Yeah, from looking at the code ( https://github.com/asergeyev/nradix/blob/master/tree.go#L157-L161 ) that seems to be the case.